ang/civiawards/award-creation/directives/review-panels/review-panels.directive.js

```diff
@@ -211,7 +211,9 @@
         visibilitySettings: {
           selectedApplicantStatus: reviewPanel.visibility_settings.application_status,
           anonymizeApplication: reviewPanel.visibility_settings.anonymize_application === '1',
-          tags: reviewPanel.visibility_settings.application_tags
+          tags: reviewPanel.visibility_settings.application_tags,
+          isApplicationStatusRestricted: reviewPanel.visibility_settings.is_application_status_restricted === '1',
+          restrictedApplicationStatus: reviewPanel.visibility_settings.restricted_application_status
```

Comment:
we don't need to check `=== '1'` ? why is this response different?

Reply:
The custom logic needs to be there, because certain properties like `isApplicationStatusRestricted`, `anonymize_application` are custom, so they dont support `TRUE`/ `FALSE` values. It can be fixed in backend, but @tunbola suggested to skip this for now as it might need more time to fix.